### PR TITLE
Add third-party theme buttons

### PR DESCRIPTION
**Pull request by jessecowens**
@bwmarkle

### CODE REVIEW

File: includes/builder/class-boldgrid-editor-builder.php [hunk 1]
```diff
@@ -134,6 +134,14 @@ public static function get_theme_buttons() {
 		global $boldgrid_theme_framework;
 
 		if ( empty( $boldgrid_theme_framework ) ) {
+			/**
+			 * Allow 3rd-Party Themes to add buttons
+			 *
+			 * @param array button_classes {
+			 * 	An key value array of button names and their classes
```
Comment (bwmarkle) [inline includes/builder/class-boldgrid-editor-builder.php:141]: Looks like we'll need a bit more spacing: https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-1-parameters-that-are-arrays

File: includes/builder/class-boldgrid-editor-builder.php [hunk 2]
```diff
@@ -134,6 +134,14 @@ public static function get_theme_buttons() {
 		global $boldgrid_theme_framework;
 
 		if ( empty( $boldgrid_theme_framework ) ) {
+			/**
+			 * Allow 3rd-Party Themes to add buttons
+			 *
```
Comment (bwmarkle) [inline includes/builder/class-boldgrid-editor-builder.php:139]: I see the "what", but can you add a "why"? e.g. Why do 3rd party themes need to do things differently?

File: README.md [hunk 3]
```diff
@@ -43,6 +43,12 @@ used in the WordPress page and post editor.
 ### Installing Dependencies
 Before you can use the development version of this plugin you must install the dependencies.
 
+First, install Node version 10 if necessary, and set the build to use it.
```
Comment (bwmarkle) [inline README.md:46]: This is awesome stuff!

File: includes/builder/class-boldgrid-editor-builder.php [hunk 4]
```diff
@@ -176,6 +184,16 @@ public static function get_theme_buttons() {
 			return $theme_buttons;
 	}
 
+	/**
+	 * Check if the current theme provides theme buttons
```
Comment (bwmarkle) [inline includes/builder/class-boldgrid-editor-builder.php:188]: You'll want to end the sentence with a period.

**Review by bwmarkle — CHANGES_REQUESTED** [review]

File: includes/builder/class-boldgrid-editor-builder.php (hunk 2 above)
Comment (jessecowens) [reply includes/builder/class-boldgrid-editor-builder.php:139]: @bwmarkle updated for you

**Review by bwmarkle — APPROVED** [review]

**Pull request merged**